src/Components/MainSection/TaskList/TaskList.jsx

```diff
@@ -1,7 +1,32 @@
 import './TaskList.css';
-
+import Task from './Task';
 function TaskList() {
-	return <div className="main-section-container">TaskList</div>;
+	return (
+		<div className="main-section-container">
+			<input className="input-bar" placeholder="Click to quickly add a task" />
+			<button className="btn">Add</button>
+			<div className="task-list-container">
+				<div className="task-container">
+					<h2 className="task-heading">
+						Tasks <span className="task-count">3</span>
+					</h2>
+					<Task />
```

Comment:
try to map over the task instead of repeating the component.

Reply:
Yes, I'll map over the tasks, This is just for visualising the UI